addons/mail/static/src/chatter/web/chatter_patch.js

```diff
@@ -200,18 +199,20 @@ patch(Chatter.prototype, {
                 const text = partner.email ? partner.emailWithoutDomain : partner.name;
                 return `<span class="text-muted" title="${escape(
```

Comment:
`markup()` should be as close as possible to code production, so when doing the return here.